src/utilities/getRatesFromLS.js

```diff
@@ -0,0 +1,26 @@
+export const getRatesFromLS = () => {
+  const savedRates = localStorage.getItem('savedRates');
+
+  if (savedRates) {
+    try {
+      const savedRatesJSON = JSON.parse(savedRates);
+
+      if (savedRatesJSON) {
+        const savedTimestamp = new Date(savedRatesJSON.timestamp);
+        const currentTimestamp = new Date();
+
+        const timeDifference =
+          currentTimestamp.getTime() - savedTimestamp.getTime();
+        const oneHour = 3600000;
+
```

Comment:
Hej, możesz także używać notacji z terenary operator:
return timeDifference < oneHour ? savedRatesJSON.rates : null;

Reply:
Faktycznie lepiej to wygląda, dzięki :)